README.md

```diff
@@ -80,7 +80,7 @@ IMAGE_TAG - image tag of the container image to be deployed as SBS server. Use "
 GITHUB_KEY_FILE - Private key path to access your GitHub repo.
 GITHUB_URL - GitHub URL.
 GITHUB_BRANCH - GitHub branch name.
-CONTAINER_NAME - Name of the Hyper Protect Virtual Servers instance which you want to create on cloud. This name can be different than the name which you use on cloud.
+CONTAINER_NAME - Name of the Hyper Protect Virtual Servers instance which you want to create on cloud. This name can be different than the name which you use on cloud. You can choose any name for the container.
```

Comment:
"Any name" is too general. Since CONTAINER_NAME is used as a part of a certificate file name (i.e. CONTAINER_NAME+'-'+uuid), it needs to be a valid string as a file name.
Perhaps, "The name is used as a part of a certificate file name. You can choose any valid string as a file name".

Reply:
Thank you very much @moriohara for your feedback. I have added both of them.